src/mscorlib/src/System/AppDomainSetup.cs

```diff
@@ -196,11 +185,7 @@ public String TargetFrameworkName
         {
             get
             {
-                return _TargetFrameworkName;
-            }
-            set
-            {
-                _TargetFrameworkName = value;
+                return null;
```

Comment:
This cleanup can go one level up to AppContext.

There is inconsistency between CoreCLR and CoreRT actually: In CoreCLR, AppContext.TargetFrameworkName will always return null. On CoreRT/ProjectN, it will return attribute value from main exe https://github.com/dotnet/corert/blob/ff7eba504516fa0bc0a50357ad559c19b45d1d51/src/System.Private.CoreLib/src/System/AppContext.cs#L45